modules/management/namespace-credential-provisioner/scripts/reconcile.sh

```
@@ -0,0 +1,456 @@
+kind: Secret
+metadata:
+  name: ${secret_name}
+  namespace: ${FLEET_DEFAULT}
+  annotations:
+    v2prov-authorized-secret-deletes-on-cluster-removal: "true"
+    v2prov-secret-authorized-for-cluster: "${cluster_name}"
+    platform.wso2.com/credential-source-namespace: "${vm_namespace}"
+type: secret
+stringData:
+  credential: |
+$(echo "$kubeconfig" | sed 's/^/    /')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)